components/TextEditor/stylesheets/QuillTextEditor.css

```diff
@@ -2,7 +2,19 @@
   border: none !important;
 }
 .ql-editor.ql-blank::before {
-  content: none;
+  left: 0 !important;
+}
+
+.ql-snow .ql-editor blockquote {
+  padding-left: 10px !important;
+}
+
+.ql-editor.peer-review.ql-blank::before {
+  top: 25px !important;
```

Comment:
can we make sure these don't cause unintended consequences? sometimes adding to this css file causes problems

Reply:
Yes, made sure!